### PR TITLE
Fix abstract reference not getting link

### PR DESCRIPTION
**Pull request by calvinlu3**

### CODE REVIEW

File: src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx [hunk 1]
```diff
@@ -12,14 +12,16 @@ export default class RefComponent extends React.Component<{
   componentType: 'tooltip' | 'linkout';
 }> {
   render() {
-    const parts = this.props.content.split(/pmid|nct|abstract/i);
+    const parts = this.props.content.split(/pmid|nct|abstract:/i);
```
Comment (zhx828) [inline src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx:15]: This will prevent abstract without colon to be recognized.

Comment (calvinlu3) [reply src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx]: Made a change, let me know what you think.

File: src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx [hunk 2]
```diff
@@ -18,8 +18,17 @@ export default class RefComponent extends React.Component<{
...
+    // when there's the case that they also contain the string 'Abstract'
+    // Example :
+    //     (Abstract: Fakih et al. Abstract# 3003, ASCO 2019.)
+    const abstractParts = parts
+      .slice(1)
+      .join('Abstract')
+      .split(/(?=http)/i);
+    const isAbstract =
+      !(abstractParts.length < 2) &&
+      this.props.content.toLowerCase().includes('abstract');
```
Comment (zhx828) [inline src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx:31]: Do we still need this check?

Comment (calvinlu3) [reply src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx]: Yeah makes sense to remove `this.props.content.toLowerCase().includes('abstract')` check.

**Review by zhx828 — APPROVED** [review]
👍🏼

**Pull request merged**